src/pages/Explorer/Explorer/explorer.module.scss

```diff
@@ -0,0 +1,242 @@
+@import '~@santiment-network/ui/mixins';
+
+.buttonsContainer {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+}
+
+.buttons {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+}
+
+.topicon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    font-family: 'Proxima Nova';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;    
+    color: #7A859E;
+
+    padding: 12px 16px 12px 12px;
+    height: 40px;
+    border: 1px solid var(--mystic);
+    box-sizing: border-box;
+    border-radius: 100px;
+    
+    &:not(:last-child) {
+        margin-right: 8px;
+    }
+
+    &:hover, &.active {
```

Comment:
Use `san-webkit`'s button class like [`btn-2`](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L35)

Reply:
also here we have `border-radius` conflict